src/transformers/pipelines/__init__.py

```diff
@@ -212,6 +214,15 @@
         },
         "type": "multimodal",
     },
+    "document-question-answering": {
+        "impl": DocumentQuestionAnsweringPipeline,
+        "pt": (AutoModelForDocumentQuestionAnswering,) if is_torch_available() else (),
+        "tf": (),
+        "default": {
+            "model": {"pt": ("dandelin/vilt-b32-finetuned-vqa", "4355f59")},  # TODO
```

Comment:
To be updated

Reply:
I left it as the `impira/` example but per your comment above, am happy to change it if you have an alternative in mind.